lib/slide_rule/distance_calculators/day_of_year.rb

```diff
@@ -3,15 +3,15 @@ module DistanceCalculators
     class DayOfYear
       DAYS_IN_YEAR = 365
 
-      def calculate(date_1, date_2)
+      def calculate(date_1, date_2, options={})
         date_1 = cleanse_date(date_1)
         date_2 = cleanse_date(date_2)
-
+        threshold = options[:threshold] || DAYS_IN_YEAR
```

Comment:
I went the more general route, that way the options established by the client would be consistent. However, that might not be the best approach. I'm definitely open to specific options.